packages/extensions-sdk/src/cli/commands/add.ts

```diff
@@ -42,7 +42,8 @@ export default async function add(): Promise<void> {
 
 	try {
 		const extensionManifestFile = await fse.readFile(packagePath, 'utf8');
-		extensionManifest = ExtensionManifest.passthrough().parse(JSON.parse(extensionManifestFile));
+		extensionManifest = JSON.parse(extensionManifestFile);
+		ExtensionManifest.passthrough().parse(extensionManifest);
```

Comment:
We no longer need to passthrough unknown keys 👍 

```suggestion
		ExtensionManifest.parse(extensionManifest);
```